<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>depositWindow</class>
 <widget class="QDialog" name="depositWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="calculateButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Расчёт</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>90</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма вклада</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>67</y>
     <width>115</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Срок размещения</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>122</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Процентная ставка</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="initialSum">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>36</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="term">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>91</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>279</y>
     <width>147</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Периодичность выплат</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>177</y>
     <width>166</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taxRate">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>248</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>224</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Налоговая ставка</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="capitalization">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>221</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Есть/Нет</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="interestRate">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>146</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="frequency">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>301</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12</string>
    </property>
   </item>
  </widget>
  <widget class="QTableWidget" name="depositTable">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>291</width>
     <height>169</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>30</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Таблица пополнений</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Таблица снятий</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="withdrawalTable">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>291</width>
     <height>169</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalInterest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>390</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма налога</string>
   </property>
  </widget>
  <widget class="QLabel" name="finalBalancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Итоговая сумма</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="taxAmount">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="finalBalance">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addDepositButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: black;
    background-color: rgb(85, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Добавить пополнение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addWithdrawalButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>500</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: black;
    background-color: rgb(85, 255, 255);
}</string>
   </property>
   <property name="text">
    <string>Добавить снятие</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="depositMonth">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>250</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>230</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Месяц</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>270</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="depositAmount">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>290</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>500</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Месяц</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>540</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Сумма</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="withdrawalAmount">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>560</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="withdrawalMonth">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>520</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Результаты по месяцам</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="resultTable">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>591</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="removeDepositButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QToolButton" name="removeWithdrawalButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>540</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
